src/app/api/v1/endpoints/suggestions.py

```
@@ -0,0 +1,24 @@
+router = APIRouter()
+
+
+@router.post(
+    "",
+    response_model=SuggestionTriggerResponse,
+    summary="Trigger AI suggestion jobs",
+    dependencies=[Depends(require_api_key)],
+)
+async def trigger_suggestions(
+    requests: list[SuggestionTriggerRequest],
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.15.20)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)